<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Гулов 4031 Сервис техобслуживания</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>681</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Вид техобслуживания</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>30</y>
       <width>661</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Ежедневное обслуживание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string> Осуществляемые работы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="text1">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Первое обслуживание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string> Осуществляемые работы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="text3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>Сезонное обслуживание</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string> Осуществляемые работы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="text2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>681</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>История записей</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>201</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Список записей работ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="list"/>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>441</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Номер записи</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="number"/>
      </item>
      <item>
       <widget class="QPushButton" name="receive_2">
        <property name="text">
         <string>Получить запись </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>Данные записи (ТО)</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>258</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Имя клиента (ТО)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="name"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Модель транспортного средтства</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="brand"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Адресс проживания клиента</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="street"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Дата прохождения ТО</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="date"/>
      </item>
      <item>
       <widget class="QPushButton" name="send">
        <property name="text">
         <string>Добавить запись</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="receive">
        <property name="text">
         <string>Удалить запись</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
